rotkehlchen/tasks/manager.py

```diff
@@ -769,6 +772,22 @@ def _maybe_update_owned_assets(self) -> Optional[list[gevent.Greenlet]]:
             user_db=self.database,
         )]
 
+    def _maybe_update_aave_v3_underlying_assets(self) -> Optional[list[gevent.Greenlet]]:
+        """
+        This function runs the logic to query the aave v3 contracts to get all the
+        underlying assets supported by them and save them in the globaldb.
+        """
+        if should_run_periodic_task(self.database, DBCacheStatic.LAST_AAVE_V3_ASSETS_UPDATE, AAVE_V3_ASSETS_UPDATE) is False:  # noqa: E501
+            return None
+
+        return [self.greenlet_manager.spawn_and_track(
+            after_seconds=None,
+            task_name='Update underlying assets in globaldb',
```

Comment:
```suggestion
            task_name='Update aave v3 underlying assets in globaldb',
```

Though I can imagine in the future this may be more generic. For now keep it accurate